### DISCUSSION TITLE
FP16 runs slower than FP32 on V100?

### DISCUSSION THREAD

**Discussion by dustinjoe** [openai/whisper#817] [Q&A]
Hi, I am trying to run a little profiling measurement on Whisper on a Tesla V100 GPU.
The performance of FP32 when FP16=False seems to be faster than the default option of FP16=True.
Generally seems to be about 1/4 faster.
I attach my test codes and results here(please remove the .txt in filename). Not sure anyone could give some suggestions on this? Thanks.

[test_whisper_time_batch0.sh.txt](https://github.com/openai/whisper/files/10355868/test_whisper_time_batch0.sh.txt)

[whisper_profile_time_args0.py.txt](https://github.com/openai/whisper/files/10355869/whisper_profile_time_args0.py.txt)

[whisper_profile_results0.rtf.txt](https://github.com/openai/whisper/files/10355874/whisper_profile_results0.rtf.txt)

**Comment by tnlin** [comment]
same there, can't figure out why
